vertical-pod-autoscaler/README.md

```diff
@@ -12,49 +12,113 @@ It can both down-scale pods that are over-requesting resources, and also up-scal
 
 Autoscaling is configured with a
 [Custom Resource Definition object](https://kubernetes.io/docs/concepts/api-extension/custom-resources/)
-called [VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta1/types.go).
+called [VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2/types.go).
 It allows to specify which pods should be under vertically autoscaled as well as if/how the
 resource recommendations are applied.
 
 To enable vertical pod autoscaling on your cluster please follow the installation
 procedure described below.
 
-
 # Installation
 
-### Notice on switching from alpha to beta (<0.3.0 to >=0.3.0)
+The current default version is Verical Pod Autoscaler 0.4.0
+
+**NOTE:** version 0.4 requires at least Kubernetes 1.11 to work (needs certain
+Custom Resource Definition capabilities). With older Kubernetes versions we
```

Comment:
In version 0.5 when we drop v1beta1 we won't need needs certain Custom Resource Definition capabilities and 1.11 won't be required.

Proposal:
**NOTE:** version 0.4 requires at least Kubernetes 1.11 to work (needs certain Custom Resource Definition capabilities to support multiple API versions). When deprecated v1beta1 API is removed requirement of Kubernetes in version at least 1.11 may be obsolete.

Reply:
I'm not sure about that. There is better validation coming to CRDs in 1.12 and in general a lot of capabilities being added that we will benefit greatly from. The current statement doesn't comment on what will happen onwards, so I would rather leave it as is.

Reply:
OK. Fair.